packages/core/src/classes.ts

```diff
@@ -271,6 +271,7 @@ export interface Space extends Doc {
   private: boolean
   members: Arr<Ref<Account>>
   archived: boolean
+  icon?: Asset
```

Comment:
We do not need it for generic spaces.

Reply:
We discussed it with Denis, and he told me that the easiest way to add a custom icon is to create this field in Space interface. But there is one more way -- to create a mixin for Space which will show some presenter with the custom icon. So, is it better to use 2nd way?

Reply:
We should not mix Core and UI in one.

Reply:
Now it is used only for teams but in future we will have to use it for userpics in direct messages

Reply:
Added a presenter component for Team